server/config/config.go

```diff
@@ -71,10 +73,12 @@ type Config struct {
 	Log     log.Config `toml:"log" env:"LOG"`
 	EtcdLog log.Config `toml:"etcd-log" env:"ETCD_LOG"`
 
-	GrpcHandleTimeoutMs int64 `toml:"grpc-handle-timeout-ms" env:"GRPC_HANDLER_TIMEOUT_MS"`
-	EtcdStartTimeoutMs  int64 `toml:"etcd-start-timeout-ms" env:"ETCD_START_TIMEOUT_MS"`
-	EtcdCallTimeoutMs   int64 `toml:"etcd-call-timeout-ms" env:"ETCD_CALL_TIMEOUT_MS"`
-	EtcdMaxTxnOps       int64 `toml:"etcd-max-txn-ops" env:"ETCD_MAX_TXN_OPS"`
+	GrpcHandleTimeoutMs     int64 `toml:"grpc-handle-timeout-ms" env:"GRPC_HANDLER_TIMEOUT_MS"`
+	GrpcOperationBurstCount int   `toml:"grpc-operation-burst-count" env:"GRPC_OPERATION_BURST_COUNT"`
```

Comment:
Use a separate configuration `FlowLimiterConfig` like `log.Config`, and provide `Enable` config for flow limiter.